### ISSUE TITLE
tabular aligning on first double backslash instead of last

### ISSUE THREAD

**Issue by hudcap** [cmhughes/latexindent.pl#179]
If there is a table with a `\shortstack{}` in it, the alignment happens based on the `\\` in the `shortstack` command instead of the `\\` at the end of the row.
MWE:
Table after being formatted:

    \begin{tabular}{lc}
        Name & \shortstack{Hi \\ Lo} \\
        Foo  & Bar            \\
    \end{tabular}

What it should look like:

    \begin{tabular}{lc}
        Name & \shortstack{Hi \\ Lo} \\
        Foo  & Bar                   \\
    \end{tabular}

Thanks for creating this awesome program!

**Comment by cmhughes** [comment]
Many thanks. This is as expected, but I'm happy to look into an option to change it.

**Comment by cmhughes** [comment]
Ps: I don't think the shortstack command has anything to do with this, I think it's the multiple \\

**Comment by hudcap** [comment]
Thank you.
Yes, I agree that it's because of the `\\`, but I just wanted to explain why I would have one in middle of a row.

**Comment by cmhughes** [comment]
As of https://github.com/cmhughes/latexindent.pl/commit/4394f8abec8a0dd37b43c2cb2f4f04d213dc9a41 this feature is implemented in `develop` and will be part of the next release. 

demonstration
-
If you start with your original file
```
\begin{tabular}{lc}
	Name & \shortstack{Hi \\ Lo} \\
	Foo  & Bar            \\
\end{tabular}
```
and specify the following YAML:
```
lookForAlignDelims:
   tabular: 
      alignFinalDoubleBackSlash: 1
```
then you receive the following output:
```
\begin{tabular}{lc}
	Name & \shortstack{Hi \\ Lo} \\
	Foo  & Bar                   \\
\end{tabular}
```

The documentation is updated to detail this feature. 

Thanks again; I'll close this issue once I've published the next release (soon, hopefully).

**Comment by cmhughes** [comment]
Implemented in `Version 3.8`, https://github.com/cmhughes/latexindent.pl/pull/190
